src/Ryujinx.Graphics.Vulkan/VulkanRenderer.cs

```diff
@@ -402,6 +422,9 @@ private unsafe void LoadFeatures(uint maxQueueCount, uint queueFamilyIndex)
                 properties.Limits.FramebufferDepthSampleCounts &
                 properties.Limits.FramebufferStencilSampleCounts;
 
+            //Temporarily disable this can be added back at a later date, make it easy to re-enable. 
```

Comment:
```suggestion
            // Temporarily disable this, can be added back at a later date, make it easy to re-enable. 
```
Maybe explain why it needs to be disabled?